features/step_definitions/smokey_steps.rb

```diff
@@ -31,7 +31,7 @@
 
   page.driver.browser.agent.add_auth(base_host, ENV['AUTH_USERNAME'], ENV['AUTH_PASSWORD'])
 
-  visit url
+  @response = get_request(url, default_request_options)
```

Comment:
This breaks several of the other tests (eg `mainstream_publishing_tools.feature`).  `visit` is a capybara method that makes a request, and then sets up the response on the `page` variable.  Later steps then rely on this being populated to make assertions against etc.  Swapping this out for a basic rest-client request causes all the later assertions to fail.